content/news/XX-XX-XXXX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,85 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+
+## Mixxx 2.4
+
+Mixxx 2.4 is our next major release shipping the work of 2 years continuous development on features, code refacturing and bug fixing.
```

Comment:
```suggestion
Mixxx 2.4 is our next major release shipping the work of 2 years continuous development on new features, code refactoring and other improvements.
```
I would like to have the bug fixes not emphasized here, because all important have gone to 2.3 branch which is of cause included.

Reply:
Was changed already